.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
             "args": ["--sync", "all", "--debugpath", "${input:debugPath}", "--debugbranch", "${input:debugBranch}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Push Binaries",
             "args": ["--push", "Binaries.zip", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Binaries",
             "args": ["--sync", "binaries", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Engine Build",
             "args": ["--sync", "engine", "--bundle", "${input:bundleSelection}", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Generate DDC",
             "args": ["--sync", "ddc", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Update Engine Version to Latest",
```

Comment:
```
11-May-20 13:25:09 [INFO ]  gs://engine.projectborealisgitlab.site/GitDependencies.exe
gs://engine.projectborealisgitlab.site/ci-engine-4.24-PB-20200403.7z
gs://engine.projectborealisgitlab.site/ci-engine-4.24-PB-20200411.7z
gs://engine.projectborealisgitlab.site/ci-engine-4.24-PB-20200412.7z
gs://engine.projectborealisgitlab.site/ci-engine-symbols-4.24-PB-20200403.7z
gs://engine.projectborealisgitlab.site/ci-engine-symbols-4.24-PB-20200411.7z
gs://engine.projectborealisgitlab.site/ci-engine-symbols-4.24-PB-20200412.7z
gs://engine.projectborealisgitlab.site/editor-4.24-PB-20200403.7z
gs://engine.projectborealisgitlab.site/editor-4.24-PB-20200411.7z
gs://engine.projectborealisgitlab.site/editor-4.24-PB-20200412.7z
gs://engine.projectborealisgitlab.site/editor-4.24-PB-20200510.7z
gs://engine.projectborealisgitlab.site/editor-symbols-4.24-PB-20200403.7z
gs://engine.projectborealisgitlab.site/editor-symbols-4.24-PB-20200411.7z
gs://engine.projectborealisgitlab.site/editor-symbols-4.24-PB-20200412.7z
gs://engine.projectborealisgitlab.site/editor-symbols-4.24-PB-20200510.7z
gs://engine.projectborealisgitlab.site/engine-4.22-PB-20190531.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191001.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191006.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191010.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191013.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191022.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191027.7z
gs://engine.projectborealisgitlab.site/engine-4.23-PB-20191102.7z
gs://engine.projectborealisgitlab.site/engine-4.24-PB-20200304.7z
gs://engine.projectborealisgitlab.site/engine-4.24-PB-20200305.7z
gs://engine.projectborealisgitlab.site/engine-4.24-PB-20200311.7z
gs://engine.projectborealisgitlab.site/engine-4.24-PB-20200323.7z
gs://engine.projectborealisgitlab.site/engine-4.24-PB-20200324.7z
gs://engine.projectborealisgitlab.site/engine-4.24-PB-20200328.7z
gs://engine.projectborealisgitlab.site/symbols-4.22-PB-20190531.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191001.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191006.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191010.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191013.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191022.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191027.7z
gs://engine.projectborealisgitlab.site/symbols-4.23-PB-20191102.7z
gs://engine.projectborealisgitlab.site/symbols-4.24-PB-20200304.7z
gs://engine.projectborealisgitlab.site/symbols-4.24-PB-20200305.7z
gs://engine.projectborealisgitlab.site/symbols-4.24-PB-20200311.7z
gs://engine.projectborealisgitlab.site/symbols-4.24-PB-20200323.7z
gs://engine.projectborealisgitlab.site/symbols-4.24-PB-20200324.7z
gs://engine.projectborealisgitlab.site/symbols-4.24-PB-20200328.7z

['editor-4.24-PB-20200403', 'editor-4.24-PB-20200411', 'editor-4.24-PB-20200412', 'editor-4.24-PB-20200510']
11-May-20 13:25:09 [INFO ]  Successfully changed engine version as 4.24-PB-20200510
```

It shouldn't redirect gcloud command's output to console

Reply:
Should be fixed now